src/storage/mvcc/reader/reader.rs

```diff
@@ -470,6 +476,8 @@ impl<S: EngineSnapshot> MvccReader<S> {
                 .prefix_seek(self.scan_mode.is_none())
                 .scan_mode(self.get_scan_mode(true))
                 .range(self.lower_bound.clone(), self.upper_bound.clone())
+                // `hint_min_ts` is only meaningful when it's the `commit_ts`.
```

Comment:
What does this comment mean? I cannot understand it...

Reply:
I saw this line of comment: https://github.com/tikv/tikv/blob/master/src/storage/mvcc/reader/scanner/mod.rs#L323

Is this still correct or did I misunderstand it?

Reply:
Ah, yeah. Write CF records commit ts range in its properties. I thought it's commented above the `hint_min_ts` field in the iterator and you were expressing something different.

You can say `hint_min_ts` filters data by commit_ts. "only meaningful when it's the `commit_ts`" is confusing.